modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -28,16 +29,15 @@
  */
 public class LocalStackContainer extends GenericContainer<LocalStackContainer> {
 
-    public static final String VERSION = "0.8.6";
 
     private final List<Service> services = new ArrayList<>();
 
     public LocalStackContainer() {
-        this(VERSION);
+        this(TestcontainersConfiguration.getInstance().getLocalStackImage().split(":")[1]);
     }
 
     public LocalStackContainer(String version) {
-        super("localstack/localstack:" + version);
+        super(TestcontainersConfiguration.getInstance().getLocalStackImage().split(":")[0] + ":" + version);
```

Comment:
if the user specifies the version, we should use the official image

Reply:
This would use the default version, unless the user has already overridden the property.

Reply:
> This would use the default version, unless the user has already overridden the property.

Yes.

However, we should use the official version, otherwise it breaks the expectation if the overridden image does not have that image tagged